geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseDataStore.scala

```diff
@@ -63,6 +66,8 @@ class HBaseDataStore(val connection: Connection, override val config: HBaseDataS
   def applySecurity(queries: Iterable[org.apache.hadoop.hbase.client.Query]): Unit =
     authOpt.foreach(a => queries.foreach(_.setAuthorizations(a)))
 
+  override def dispose(): Unit = try { super.dispose() } finally { con.close() }
```

Comment:
Huh, we didn't have a dispose before?  *blinks*

Reply:
yep, b/c the connection was shared so we didn't close it